### ISSUE TITLE
Build fails since security improvements in git protocole

### ISSUE THREAD

**Issue by Szewcson** [mirage/qubes-mirage-firewall#139]
When try to build got:
```
Step 2/9 : RUN cd ~/opam-repository && git fetch origin master && git reset --hard 479a47921a489d11833e03cf949bfb612bd65e41 && opam update
 ---> Running in 4efb1f6ecce5
fatal: remote error: 
  The unauthenticated git protocol on port 9418 is no longer supported.
Please see https://github.blog/2021-09-01-improving-git-protocol-security-github/ for more information.
The command '/bin/sh -c cd ~/opam-repository && git fetch origin master && git reset --hard 479a47921a489d11833e03cf949bfb612bd65e41 && opam update' returned a non-zero code: 128
```

**Comment by Szewcson** [comment]
I was able to fix it by adding:
`RUN git config --global url."https://".insteadOf git://`
before git fetch. But there is later a problem with gathering sources by opam:
```
[ERROR] The sources of the following couldn't be obtained, aborting:
          - ocaml-freestanding.0.6.6: Bad checksum

make: *** [Makefile:15: depend] Error 40
```
Maybe it's time to update building script to use more up to date version of opam-repository and update shasum to new one.

**Comment by hannesm** [comment]
fixed by merge of #140 -- please re-open (or open a new issue) if this is still a problem for you.
